images/management-api-for-apache-cassandra/main.tf

```diff
@@ -1,17 +1,35 @@
 terraform {
   required_providers {
-    oci = { source = "chainguard-dev/oci" }
+    oci  = { source = "chainguard-dev/oci" }
+    apko = { source = "chainguard-dev/apko" }
   }
 }
 
+// Create a dummy config that includes 'cassandra;, so we can extract the actual cassandra version below.
+data "apko_config" "for-getting-cassandra-version" {
+  config_contents = jsonencode({
+    contents = {
+      packages = ["cassandra"]
+    }
+  })
+}
+
+// Extract the cassandra version from the dummy config.
+locals {
```

Comment:
This image requires 'CASSANDRA_VERSION' to be set as env var. Rather than hard code this, we grab it from the package. This isn't pretty, but we do similar in other places, so it's not new, and it means no more hard-coding CASSANDRA_VERSION